files/image_config/interfaces/interfaces.j2

```diff
@@ -53,29 +53,36 @@ iface {{ interface['attachto'] }} {{ 'inet' if interface['addr'] | ipv4 else 'in
     netmask {{ interface['mask'] }}
 #
 {% endfor %}
+{% if minigraph_vlans.keys() | length %}
+# "|| true" is added to suppress the error when interface is already a member of VLAN
+{% endif %}
 {% for vlan in minigraph_vlans.keys()|sort %}
 {% for member in minigraph_vlans[vlan]['members'] %}
 auto {{ member }}
 allow-hotplug {{ member }}
 iface {{ member }} inet manual
     pre-up ifconfig {{ member }} up
-    post-up brctl addif {{ vlan }} {{ member }}
+    post-up brctl addif {{ vlan }} {{ member }} >/dev/null 2>&1 || true
```

Comment:
is it necessary to redirect the standout to /dev/null? would it be bettr if we can see the error message? in case there are some other errors?